src/openrct2/interface/Cursors.h

```diff
@@ -11,42 +11,42 @@
 
 #include "../common.h"
 
-enum CURSOR_ID
```

Comment:
Please change this to uint8_t, and define the Undefined cursor as 0xFF.

Your method is understandable and not wrong, but we almost always do it as I described above, so I prefer being consistent.